android-radar/build.gradle

```diff
@@ -53,7 +30,33 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 }
 
-group = publishedGroupId
-version = libraryVersion
-apply from: 'https://raw.githubusercontent.com/cedexis/JCenter/master/installv1.gradle'
-apply from: 'https://raw.githubusercontent.com/cedexis/JCenter/master/bintrayv1.gradle'
\ No newline at end of file
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                pom {
+                    name = 'Android Radar Library'
+                    description = 'An open source Radar client for Android'
+                    url = 'https://github.com/cedexis/AndroidRadar'
+                    licenses {
+                        license {
+                            name = 'The MIT License (MIT)'
+                            url = 'http://opensource.org/licenses/MIT'
+                        }
+                    }
+                    developers {
+                        developer {
+                            id = 'cedexis'
+                            name = 'Cedexis'
+                            email = 'support@cedexis.com'
```

Comment:
I wonder if this email is still valid and if anyone is monitoring it.

Reply:
@ikalligeros Yes this still replies, it redirects the sender to citrix support.

Reply:
I have removed for now the email, in lack of a good alternative. Users can submit tickets for now.